packages/fether-react/src/Send/TxForm/TxForm.js

```diff
@@ -80,6 +81,7 @@ class Send extends Component {
                           className='-sm'
                           label='To'
                           name='to'
+                          id='to'
```

Comment:
It's actually better to put `id` in all Fields, i.e. here: https://github.com/paritytech/fether/blob/master/packages/fether-ui/src/Form/Field/Field.js#L32, so that all Fields in all forms will have an id.

```jsx
<T id={input && input.name} {...input} {...otherProps} />
```